{{cookiecutter.project_slug}}/clients/react/tests/e2e/specs/test-login.cy.ts

```diff
@@ -0,0 +1,6 @@
+describe('Tests login workflow', () => {
+  it('Home page auto redirects to login', () => {
+    cy.visit('/')
+    cy.url().should('include', '/log-in')
+  })
+})
```

Comment:
This is not a real test of login. Looks like you removed the already existing test that does test login.